java/java8/Dockerfile

```diff
@@ -12,11 +12,55 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Start of Airlock configuration go/airlock/howto_docker
```

Comment:
Can we refactor this so that we don't have to repeat the same block in every Docker image?

Reply:
Let me try `INCLUDE+` https://stackoverflow.com/a/65567427

Reply:
It turned out that the URL `ar+https://us-apt.pkg.dev/projects/artifact-foundry-prod standard-debian-bullseye-3p-l1 main` differs between Ubuntu/Debian releases. A shared file doesn't work.